src/ToricVarieties/NormalToricVarieties/attributes.jl

```diff
@@ -313,6 +313,167 @@ end
 export map_from_weil_divisors_to_class_group
 
 
+@doc Markdown.doc"""
+    map_from_cartier_divisor_group_to_torus_invariant_divisor_group(v::AbstractNormalToricVariety)
+
+Computes the embedding of the group of Cartier divisors into the group of torusinvariant Weil divisors of an abstract normal toric variety `v`.
+
+# Examples
+```jdoctest
+julia> p2 = toric_projective_space(2)
+A normal toric variety corresponding to a polyhedral fan in ambient dimension 2
+
+julia> map_from_cartier_divisor_group_to_torus_invariant_divisor_group(p2)
+Map with following data
+Domain:
+=======
+Abelian group with structure: Z^3
+Codomain:
+=========
+Abelian group with structure: Z^3
+```
+"""
+function map_from_cartier_divisor_group_to_torus_invariant_divisor_group(v::AbstractNormalToricVariety)
+    # check input
+    if hastorusfactor(v)
+        throw(ArgumentError("Group of the torus-invariant Cartier divisors can only be computed if the variety has no torus factor."))
+    end
+    
+    # identify and format the rays
+    initial_rays = Polymake.common.primitive(pm_object(v).RAYS)
+    rays = Vector{fmpz}[]
+    for i in 1:size(initial_rays)[1]
+        push!(rays, [fmpz(c) for c in initial_rays[i,:]])
+    end
```

Comment:
Perhaps also like this?
```suggestion
    rays = [[fmpz(c) for c in initial_rays[i,:]] for i in 1:nrows(initial_rays)]
```

Reply:
Personally I really prefer `nrows` / `ncols` (from Oscar) over these `size(FOO)[1]` / `size(FOO)[2]` constructs